apps/fp-frontend/src/app/components/useAppVersjonDeteksjon.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+
+const POLLING_INTERVAL_MS = 10 * 60 * 1000;
+
+export const useAppVersjonDeteksjon = () => {
+  const [serverVersjon, setServerVersjon] = useState<string | null>(null);
+  const [nyVersjonTilgjengelig, setNyVersjonTilgjengelig] = useState(false);
+
+  const [klientVersjon, setKlientVersjon] = useState<string | null>(null);
```

Comment:
dette er pirk på høgt nivå, men for meg hadde det vore meir naturleg at linje 7 og 9 byttar plass